src/vulkan/pipeline.h

```diff
@@ -50,8 +51,8 @@ class Pipeline {
 
   Result AddDescriptor(const BufferCommand*);
 
-  // Read back the contents of resources of all descriptors to a
-  // buffer data object and put it into buffer data queue in host.
+  /// Read back the contents of resources of all descriptors to a
```

Comment:
Nit that I usually let pass.  But this is a good time set this straight.  

Style should be "Reads ..."

From the style guide:

> These comments should be descriptive ("Opens the file") rather than imperative ("Open the file"); the comment describes the function, it does not tell the function what to do.

Reply:
done